.gitignore

```diff
@@ -282,10 +285,4 @@ cython_debug/
 
 # Emacs
 *~
-\#*\#
-
-coverage-reports/*
-
-# VS Code
-.vscode/*
-*.code-workspace
+\#*\#
```

Comment:
@marknoack Do you know what happened here? Is this an intentional change?

Reply:
It's unintentional. No idea how this code snippet was added

Reply:
I have reverted it